docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -70,23 +137,116 @@ EOF
 
 ----
 
-. Create S3 buckets
+. Create S3 bucket
 +
 [source,console]
 ----
-exo sos create "${CLUSTER_ID}-bootstrap"
-exo sos create "${CLUSTER_ID}-tf-state"
+exo storage create "sos://${CLUSTER_ID}-bootstrap"
 
 export TF_VAR_bootstrap_bucket="https://sos-${EXOSCALE_REGION}.exo.io/${CLUSTER_ID}-bootstrap/"
 ----
 
+=== Upload Red Hat CoreOS image
+
+. Fetch and convert the latest Red Hat CoreOS image
++
+[source,console]
+----
+RHCOS_VERSION="4.7.7"
+
+curl "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/${RHCOS_VERSION}/rhcos-${RHCOS_VERSION}-x86_64-openstack.x86_64.qcow2.gz" | gunzip > rhcos-${RHCOS_VERSION}.qcow2
+
+virt-edit -a rhcos-${RHCOS_VERSION}.qcow2 \
+  -m /dev/sda3:/ /loader/entries/ostree-1-rhcos.conf \
+  -e 's/openstack/exoscale/'
+
+exo storage upload rhcos-${RHCOS_VERSION}.qcow2 "sos://${CLUSTER_ID}-bootstrap" --acl public-read
+
+exo vm template register "rhcos-${RHCOS_VERSION}" \
+  --checksum $(md5sum rhcos-${RHCOS_VERSION}.qcow2 | awk '{ print $1 }') \
+  --boot-mode uefi \
+  --disable-password \
+  --username core \
+  --description "Red Hat Enterprise Linux CoreOS (RHCOS) ${RHCOS_VERSION}" \
+  --url "https://${AWS_S3_ENDPOINT}/${CLUSTER_ID}-bootstrap/rhcos-${RHCOS_VERSION}.qcow2"
+
+exo storage delete "sos://${CLUSTER_ID}-bootstrap/rhcos-${RHCOS_VERSION}.qcow2"
+
+export TF_VAR_rhcos_template="rhcos-${RHCOS_VERSION}"
+----
+
+. Upload Ursula Debian package to bootstrap bucket
++
+[source,console]
+----
+exo storage upload /path/to/ursula.deb "sos://${CLUSTER_ID}-bootstrap/ursula.deb" --acl public-read
+----
+
+
+=== Set secrets in Vault
+
+.Connect with Vault
+[source,console]
+----
+export VAULT_ADDR=https://vault-prod.syn.vshn.net
+vault login -method=ldap username=<your.name>
+----
+
+.Store various secrets in Vault
+[source,console]
+----
+# Set the exoscale.ch access secrets
+vault kv put clusters/kv/${TENANT_ID}/${CLUSTER_ID}/exoscale \
+  api_key=${EXOSCALE_API_KEY} \
+  api_secret=${EXOSCALE_API_SECRET} \
+  s3_access_key=${EXOSCALE_API_KEY} \
+  s3_secret_key=${EXOSCALE_API_SECRET}
```

Comment:
Why is this necessary?

Reply:
The S3 credentials are also in the cloudscale instructions. Could it be for K8up cluster-backup?

Reply:
The S3 ones are probably for K8up, but we definitely don't need `api_key` and `api_secret` as those are used by cloudscale-csi.